### PR TITLE
{vis}[foss/2022a] yt v4.3.0 

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

Requires:
- [x] PR #19756

### CODE REVIEW

**Comment by sassy-crick** [comment]
I am having problems to get that compilation working and the error messages are, well, cryptic to me. 
The full log can be found [here](https://gist.github.com/sassy-crick/59098cedb64e7e689c83e2766ebd888f)
I would appreciate some help. thanks

**Review by bedroge — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/y/yt/yt-4.3.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+dependencies = [
+    ('Python', '3.10.4'),
+    ('astropy', '5.1.1'),
+    ('h5py', '3.7.0'),
+    ('SciPy-bundle', '2022.05'),
+    ('IPython', '8.5.0'),
+    ('matplotlib', '3.5.2'),
+    ('Pillow', '9.1.1'),
+    ('tqdm', '4.64.0'),
+    ('sympy', '1.11.1'),
```
[the remark this answers is not in the document]
Comment (sassy-crick) [reply easybuild/easyconfigs/y/yt/yt-4.3.0-foss-2022a.eb]: Done. Thanks for finding this. I had some problems to get the required versions correctly with that software. Also, if there are two versions in the same toolchain I usually go for the newest one.

File: easybuild/easyconfigs/y/yt/yt-4.3.0-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    }),
+    (name, version, {
+        'checksums': ['cdcab7c07d8c46fe87d94fcec4bbac750394aa40b88cb7381abc843dcdce30ee'],
+    }),
+]
+
+sanity_check_commands = [
+    "python -c 'import yt'",
+]
+
```
[the remark this answers is not in the document]
Comment (sassy-crick) [reply easybuild/easyconfigs/y/yt/yt-4.3.0-foss-2022a.eb]: Done. Thanks for finding that.

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node63 - Linux Rocky Linux 8.9, x86_64, AMD EPYC 7763 64-Core Processor (zen3), Python 3.6.8
See https://gist.github.com/bedroge/5aeca81d16227f95bdbad2f39baf1821 for a full test report.

**Review by bedroge — APPROVED** [review]
Lgtm

**Comment by bedroge** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**